utils/validate-facility-ids/index.ts

```diff
@@ -0,0 +1,32 @@
+import { difference } from 'lodash';
+import { SqlDbDataSource } from '@ukef/dtfs2-common/sql-db-connection';
+import { mongoDbClient } from './database-client';
+import { getUniqueFeeRecordFacilityIds, getUniqueTfmFacilitiesUkefFacilityIds } from './helpers';
+
+const run = async (): Promise<void> => {
+  const dataSource = await SqlDbDataSource.initialize();
+  try {
```

Comment:
Should we add a `console.info` here? 

Can we add documentation?

Reply:
Done